x509/cert_validations_test.go

```diff
@@ -158,6 +160,39 @@ var codeSigningLeafInvalidPathLenPem = "-----BEGIN CERTIFICATE-----\n" +
 	"L0Z9ZU3NwwFAh1wpSsoQR4pPN+ZkVj5Irr/KWFk=\n" +
 	"-----END CERTIFICATE-----"
 
+var openSSLMinimumPem = "-----BEGIN CERTIFICATE-----\n" +
```

Comment:
nit: we can use backquotes (see [string literals](https://go.dev/ref/spec#String_literals)) or embedding files (see [embed](https://pkg.go.dev/embed)).